src/components/backend-ai-session-view.ts

```diff
@@ -464,9 +471,8 @@ export default class BackendAiSessionView extends BackendAIPage {
                   </mwc-menu>
                 </div>
               ` : html``}
-            </div>
             <div class="horizontal layout flex end-justified" style="margin-right:20px;">
-            <backend-ai-session-launcher location="session" id="session-launcher" ?active="${this.active === true}" isSupportingFab></backend-ai-session-launcher>
+              <backend-ai-session-launcher location="session" id="session-launcher" ?active="${this.active === true}" isSupportingFab></backend-ai-session-launcher>
             </div>
```

Comment:
@ggstone0523 Let's remove `isSupportingFab` attribute in Line 475. It seems like a legacy code and we are no longer using that attribute anymore.